test/test_session_params.cpp

```diff
@@ -58,7 +58,7 @@ namespace
 
 	struct custom_plugin : plugin
 	{
-		void added(session_handle h) override
+		void added(session_handle& h) override
```

Comment:
I wanted to change it to keep it similar (in style) to when `peer_connection_handle` is passed. I will put the `const`.